packages/zone.js/test/browser/browser.spec.ts

```diff
@@ -1035,6 +1037,42 @@ describe('Zone', function() {
            button.removeEventListener('click', listener);
          }));
 
+      describe('passiveEvents', () => {
+        let logs: string[] = [];
+        const listener = (e: Event) => {
+          logs.push(e.defaultPrevented.toString());
```

Comment:
Instead of pushing `true`/`false` which is hard to understand in the assertion, could we push text
```suggestion
          logs.push(e.defaultPrevented ? 'defaultPrevented' : 'default will run');
```

Reply:
Yeah, this is better, updated.